lib/manager/leiningen/extract.ts

```diff
@@ -96,7 +95,7 @@ export function extractFromVectors(
 }
 
 function extractLeinRepos(content: string): string[] {
-  const result = [CLOJARS_REPO, MAVEN_REPO];
```

Comment:
Previously, it looks like these were prepended to any configured repositories. But now they are `defaultRegistryUrls`. This is a functional change, right?

Reply:
It should work fine after merge, affecting only order of HTTP request